registry/txt.go

```diff
@@ -135,6 +144,23 @@ func (im *TXTRegistry) Records(ctx context.Context) ([]*endpoint.Endpoint, error
 				ep.Labels[k] = v
 			}
 		}
+		// Handle the migration of TXT records created before the new format (introduced in v0.12.0)
+		if len(txtRecordsMap) > 0 {
+			if plan.IsManagedRecord(ep.RecordType, im.managedRecordTypes) {
+				// Get desired TXT records and detect the missing ones
+				desiredTXTs := im.generateTXTRecord(ep)
+				missingDesiredTXT := []*endpoint.Endpoint{}
```

Comment:
For consistency with "desiredTXTs" on the line above:
```suggestion
				missingDesiredTXTs := []*endpoint.Endpoint{}
```